Tasks/Anastasia Konuyk/Task3/Task3.py

```diff
@@ -0,0 +1,40 @@
+def lenline():
+    while True:
+        try:
+            lenght = int(input("Please enter length line (more than 35 characters): "))
+            if lenght > 35:
+                return lenght
+            print(f"The {lenght} isn't correct, try again")
+        except ValueError:
+            print("ValueError")
+        continue
+
+wight = lenline()
+
+def justify(text_from_file):
+    words = list(reversed(text_from_file.split()))
+    result = []
+    while words:
+        line = [words.pop()]
+        count = len(line[0])
+        while words and count + len(words[-1]) + 1 <= wight:
+            line.append(words.pop())
+            count += len(line[-1]) + 1
+        if len(line) == 1 or not words:
+            result.extend(' '.join(line))
+        else:
+            num_spaces = wight - sum(len(word) for word in line)
+            q, r = divmod(num_spaces, len(line) - 1)
+            pads = [0 if indx == 0 else q if indx > r else q + 1 for indx in range(len(line))]
+            result.extend(' ' * pad + word for pad, word in zip(pads, line))
+        result.append('\n')
+    return ''.join(result)
+
+
+with open("text.txt", "r", encoding = "utf-8") as old:
+    text_from_file = old.read()
+    lst_final = justify(text_from_file)
```

Comment:
reading the whole file in memory is not a good approach, since it can be huge, it would be better to read and write the text in a small portions, -10 points for the not optimal solution